dev/diagrams/on-sign-out-app.md

```diff
@@ -31,7 +31,7 @@ box
 activate user
```

Comment:
can you update the TOC too?

Please download
https://marketplace.visualstudio.com/items?itemName=huntertran.auto-markdown-toc

Reply:
Handled in [fb0157c](https://github.com/ajktown/docs/pull/57/commits/fb0157cc9544f2b1a9b923a6f0f71d43bb4b3e5d)

Reply:
checked